bin/p2-inspect/main.go

```diff
@@ -59,11 +59,11 @@ func main() {
 
 	intents, _, err := store.ListPods(kp.INTENT_TREE)
 	if err != nil {
-		log.Fatalf("Could not list intent kvpairs: %s", err)
+		log.Fatalf("Could not list intent kvpairs: %s", err.(kp.KVError).UnsafeError)
```

Comment:
I'm in general OK with this, but what will happen in code paths where `err` is not a `kp.KVError`? I believe there is such a code path. See at https://github.com/square/p2/blob/master/pkg/kp/kv.go#L94 and https://github.com/square/p2/blob/master/pkg/pods/manifest.go#L81 - are you OK with that?